src/EditorFeatures/Core/ValueTracking/ValueTrackingService.cs

```diff
@@ -95,6 +89,20 @@ or ILocalSymbol
                     await progressCollector.TryReportAsync(document.Project.Solution, node.GetLocation(), symbol, cancellationToken).ConfigureAwait(false);
                 }
             }
+            else if (node is not null && symbol is null)
+            {
+                // If the correct symbol couldn't be determined but a node is available, try to let
+                // the operation collector handle the operation and report as needed
```

Comment:
not really clear when this happens.

Reply:
This was used for when the node was an argument but the symbol was unclear. The operation collector handles those well. However, the test case that was hitting this is no longer hitting it. Looking further it's because of some of the changes I made elsewhere making this no longer needed. Removing. There may still be cases where this will be needed, but not currently handled in tests.